recipe/migrations/azure_core_cpp1130.yaml

```
@@ -0,0 +1,8 @@
+__migrator:
+  build_number: 1
+  commit_message: Rebuild for azure_core_cpp 1.13.0
+  kind: version
+  migration_number: 1
+azure_core_cpp:
+- 1.13.0
```

```suggestion
- 1.13.0
azure_storage_common_cpp:
- 12.7.0
```

Proposal to combine with https://github.com/conda-forge/conda-forge-pinning-feedstock/pull/6194

---

I see. I wrongly thought that the migration would start by first migrating azure-storage-common-cpp-feedstock to use azure-core-cpp 1.13.0, ie I thought that your manual PR https://github.com/conda-forge/azure-storage-common-cpp-feedstock/pull/18 was going to be performed by the bot. Now I know. Thanks for handling it! I just merged it

---

> I wrongly thought that the migration would start by first migrating azure-storage-common-cpp-feedstock to use azure-core-cpp 1.13.0

**UPDATE:** Need to set `exclude_pinned_pkgs: false` for migration PRs to be sent to the pinned packages themselves (https://github.com/conda-forge/conda-forge-pinning-feedstock/pull/6208#issuecomment-2251423758)